spec/factories/authorizations.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: authorizations
+#
+#  id         :integer          not null, primary key
+#  user_id    :integer
+#  provider   :string
+#  uid        :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+FactoryGirl.define do
+  factory :authorization do
+    user_id 1
```

Comment:
for associations please use like `association :user`

Reply:
i don't start testing it yet. this file is generated by 'rails g' I think